src/main/java/duke/command/TaskDoneCommand.java

```diff
@@ -0,0 +1,48 @@
+package duke.command;
+
+import duke.data.Storage;
+import duke.data.TaskList;
+import duke.exception.InvalidTaskNoException;
+
+/**
+ * Represents a command that marks a task as done. A subclass of the Command class.
+ */
+public class TaskDoneCommand extends Command {
+    /** Index of the task to be deleted in the task list */
+    private int taskIndex;
+
+    /**
+     * Constructor of the class `TaskDoneCommand`.
+     *
+     * @param taskIndex Index of a task.
+     */
+    public TaskDoneCommand(int taskIndex) {
+        super("done");
```

Comment:
Perhaps you can create a private static final variable for these magic literals? I noticed this issue in other places as well.